content/docs/glossary/transactions.mdx

```diff
@@ -42,10 +42,39 @@ The memo contains optional extra information. It is the responsibility of the cl
+
+For some of these, it's important to understand a related concept: the last time an account consumed a sequence number. Both the ledger number and timestamp of the last time this occurred is tracked by the network, and these values enable some of the preconditions below.
+
+#### Time Bounds
 The optional UNIX timestamp (in seconds), determined by ledger time, of a lower and upper bound of when this transaction will be valid. If a transaction is submitted too early or too late, it will fail to make it into the transaction set. `maxTime` equal `0` means that it's not set. _We highly advise for all transactions to use time bounds, and many SDKs enforce their usage._ If a transaction doesn't make it into the transaction set, it is kept around in memory in order to be added to the next transaction set on a best-effort basis. Because of this behavior, we highly advise that all transactions are created with time bounds in order to invalidate transactions after a certain amount of time, especially if you plan to resubmit your transaction at a later time.
 
+#### Ledger Bounds
+These are like [Time Bounds](#time-bounds), except they apply to ledger numbers. With them set, a transaction will only be valid for ledger numbers that fall into the range you set. The lower bound is inclusive while the upper bound is not. If you set the upper bound to zero, this indicates that there is no upper bound.
+
+#### Minimum Sequence Number
+If this precondition is set, the transaction will only be valid when its source account's sequence number (call it `S`) is large enough. Specifically, it's valid when `S` satisfies `minSeqNum <= S < tx.seqNum`. If this precondition is omitted, the default behavior applies: the transaction's sequence number must be exactly one greater than the account's sequence number.
+
+Note that after a transaction is executed, the account will always raise its sequence number to the transaction's sequence number.
+
+#### Minimum Sequence Age
+With this precondition, the transaction is only valid after an account's last sequence number change exceeds a particular duration (expressed in seconds).
+
+With a minimum [time bound](#time-bounds), the transaction becomes valid after a particular amount of time passes unconditionally. With this precondition, the passage of time applies only to the transaction's source account: you can allow validity after an *account* has waited long enough since its last sequence number change.
+
+#### Minimum Sequence Ledger Gap
+With this precondition, the transaction is only valid after the current network ledger number meets (or exceeds) a particular gap relative to the ledger corresponding to the source account's last sequence number change.
```

Comment:
Same as above:
```suggestion
Minimum sequence ledger gap makes a transaction valid only if a duration of ledgers has passed since the last ledger that the sequence number for a source account has been operated on, either by a BumpSequence operation, or a Transaction. Note that a BumpSequence resets this period regardless of whether the sequence number of the source account is changed.
```

Reply:
I consolidated this into parent paragraph - _Validity Conditions_ - and clarify/introduce/define what it means for an account sequence number to be touched (I also think a term like `refreshed` could work).